webpack/assets/javascripts/react_app/components/notifications/notifications.test.js

```diff
@@ -5,7 +5,7 @@ import React from 'react';
 import configureMockStore from 'redux-mock-store';
 import thunk from 'redux-thunk';
 
-import { getStore } from '../../redux';
+import { generateStore } from '../../redux';
```

Comment:
whats the rational to rename the method?

Reply:
I renamed this to `generateStore` since it's actually generating a new store each time. The main application store is exported as the default but test files use `generateStore` to generate a unique store for the test suite.